src/map/clif.cpp

```diff
@@ -2783,6 +2783,18 @@ void clif_inventorylist(struct map_session_data *sd) {
 			n++;
 		}
 	}
+
+	// Send fake egg data
+	if (battle_config.feature_petevolution && sd->pd) {
+		int pos = ne * se + 4;
```

Comment:
This is causing overflowed pointer if the player has 100 items in his inventory.

Reply:
Aegis manages pet eggs by reserving a item slot for the egg when the pet is hatched, so you never could have 100 items + a pet

Reply:
This will most likely need a personalized maximum inventory slot system. Also to prepare for the inventory expansion system kRO has.